package.json

```diff
@@ -13,7 +13,9 @@
   "dependencies": {
```

Comment:
I'm just leaving this comment over here, but it is unrelated to this code. There are some minor reasons due to which the build failed. We'll require a passing build to merge this code to the `main` branch.